docs/contrib/sequences.rst

```diff
@@ -0,0 +1,79 @@
+==============
+Lazy sequences
+==============
+
+.. versionadded:: 0.12.0
+
+Sequences module contains few macros for declaring sequences that are evaluated
+only as much as the client code requests elements. Compared to generators they
```

Comment:
I know this is SUPER nit-picky...but there's a missing comma after *generators*...